### ISSUE TITLE
Globals:  Cannot read property 'suggestName' of undefined

### ISSUE THREAD

**Issue by sloosch** [Pauan/rollup-plugin-purs#11]
Having a FFI-File with:
````javascript
a = function () {}
exports.foo = a;
````
where ````a```` is declared as some global function and the purescript file:
````purescript
module Foo where

foreign import foo :: String
````
The plugin fails with "Cannot read property 'suggestName' of undefined."

In contrast the slightly modified FFI-File:
````javascript
a = function () {}
var b = a;
exports.foo = b;
````
does work.

This is especially a problem when trying to use the purescript-globals module which does export  globals by referencing global functions e.g.:
````javascript
...
exports.readFloat = parseFloat;
...
````

**Comment by Pauan** [comment]
@sloosch Hm, this may actually be expected behavior. ES6 modules do not allow for defining global variables, so `a = function () {}` is wrong. It has to be `var a = function () {}` instead.

**Comment by Pauan** [comment]
`exports.readFloat = parseFloat` should work, though, so I'll need to look into this

**Comment by Pauan** [comment]
Okay, I fixed this in version 1.0.27, thanks!

Defining global variables like `a = function () {}` may or may not work, but `exports.readFloat = parseFloat` will definitely work now.
